src/twisted/web/_element.py

```diff
@@ -181,3 +181,7 @@ def render(self, request):
         if loader is None:
             raise MissingTemplateLoader(self)
         return loader.load()
+
+
+if TYPE_CHECKING:
+    from twisted.web.template import Flattenable, Tag
```

Comment:
Curious why not move this up so you don't need to put quotes around them in the file.

Reply:
Just my 2¢ here, I'd rather not see type annotation PRs ever introduce additional runtime coupling if they can possibly avoid it. When we adopt 3.7 we can drop all the quotes with `from __future__ import annotations`

Reply:
(In the meanwhile they seem like a small price to pay to avoid having to think hard about circular dependencies or import times)

Reply:
Circular dependencies are indeed a potential issue here: `template` imports `_element`, but it does so at the end of the module, so it would probably be fine but it would be brittle.

Therefore I'd rather leave the `TYPE_CHECKING` in place. I could move the statement to the top of the file, if you think that makes it easier to read.